test/jdk/java/lang/Math/ExactArithTests.java

```diff
@@ -132,6 +132,39 @@ static void testIntegerExact(int x, int y) {
             }
         }
 
+        boolean exceptionExpected = false;
```

Comment:
Please add "divideExact" to the list of methods tested by 
static void testIntegerExact(int x, int y) 
and similar change for long.